src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -128,6 +129,7 @@ public function __construct(array $connectionOptions, array $exchangeOptions, ar
      *   * read_timeout: Timeout in for income activity. Note: 0 or greater seconds. May be fractional.
      *   * write_timeout: Timeout in for outcome activity. Note: 0 or greater seconds. May be fractional.
      *   * connect_timeout: Connection timeout. Note: 0 or greater seconds. May be fractional.
+     *   * confirm_timeout: Timeout in for confirmation, if none specified transport will not wait for message confirmation. Note: 0 or greater seconds. May be fractional.
```

Comment:
```suggestion
     *   * confirm_timeout: Timeout in seconds for confirmation, if none specified transport will not wait for message confirmation. Note: 0 or greater seconds. May be fractional.
```

I guess you need to update it in the ones above as well.

Reply:
Thanks! Fixed.